tests/foreman/ui/test_registration.py

```diff
@@ -0,0 +1,313 @@
+"""Tests for registration.
+
+:Requirement: Registration
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Registration
+
+:CaseAutomation: Automated
+
+:CaseImportance: Critical
+
+:Team: Rocket
+
+:TestType: Functional
+
+:Upstream: No
+"""
+from datetime import datetime
+
+import pytest
+from fauxfactory import gen_string
+
+from robottelo import constants
```

Comment:
you are importing all constants while 2 lines later you are re-importing specific ones. Is there a reason for it?

Reply:
It is not needed. I will update the import.